change/@fluentui-react-8d5952ad-7979-4a9c-bae9-f37a22caf598.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28439&commitId=1c25246bb115827f2fe50e2c54c470828ef83fea&env=prod&clientType=FLUENTUI)


  <details><summary><b>GroupedList</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|GroupedList.Root.click.chromium.png|87| Changed|
|GroupedList.Root - RTL.click.chromium.png|87| Changed|
|GroupedList.Root - RTL.default.chromium.png|63| Changed|
|GroupedList.Root.hover.chromium.png|109| Changed|
|GroupedList.Root.default.chromium.png|64| Changed|
|GroupedList.Root - RTL.hover.chromium.png|108| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>